.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+To see if your fork is outdated, open the fork page in GitHub. If you see the following message at the top of the page, update your fork: `This branch is <number> commits behind develop`.
+
+There are two ways to update your fork. The typical way is discussed in [GitHub documentation](https://help.GitHub.com/articles/syncing-a-fork){:target="_blank"}. Ensure you are updating your fork from the correct branch.
+
+It is also possible to use the GitHub interface to update your fork. This is referred to as a *reverse pull request*. This method has the downside of inserting unnecessary information into fork commit history.
+
+1.  Navigate to the GitHub page for your forked repository and click **New pull request**. You should see the following message:
+
+        There isn’t anything to compare.
+        magento:2.1 is up to date with all commits from <your fork>:2.1. Try switching the base for your comparison.
+
+1.  Click the base link and then click **Create pull request**.
+
+1.  Enter a name for your pull request.
+
+1.  Scroll to the bottom of the page and click **Merge pull request**.
+
+1.  Click **Confirm Merge**.
+
+## Create a pull request {#pull-request}
+To create a pull request do the following:
+
+1.  Push your changes to your forked repository on GitHub.
+
+1.  In your forked repository, click **New pull request**.
+
+1.  Be sure to create the pull request on the `develop` branch. We do not accept pull requests on other branches, like `gh-pages`.
+
+1.  Review the changes, then click **Create pull request**.
+
+1.  Fill out the form and click **Create pull request** again to submit the pull request&mdash;that’s it!
+
+<div class="bs-callout bs-callout-info" id="info" markdown="1">
+If you have not signed the [Magento Contributor Agreement](http://www.magento.com/legaldocuments/mca), the PR will provide a link to complete. All contributors are required to submit the form and agree to the terms.
+</div>
+
+## Doc templates {#templates}
+
+Edit and add content to existing pages. For new topics, we provide templates to help get you started:
+
+* [General topic template]({{ page.baseurl }}/contributor-guide/templates/basic_template.html): This is a template for writing any topic with example formats and styles.
+* Tutorial templates: These templates provide example formats and styles for step-by-step instructions (like how-tos). Templates include a [first introduction topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-first.html), [middle topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-middle.html) for each step, and a [final step topic]({{ page.baseurl }}/contributor-guide/templates/tutorial-template-last.html).
+
+### Edit metadata
+The Markdown (.md) file's {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} is a set of YAML key-value pairs. The metadata section is located at the top of each file.
```

Comment:
remove this `glossarytooltip` liquid tag

Reply:
Removed